samples/drivers/smbus/README.rst

```diff
@@ -1,7 +1,8 @@
-.. _samples_smbus_shell:
+.. zephyr:code-sample:: smbus-shell
+   :name: SMBus shell
+   :relevant-api: smbus_interface
 
-SMBus Shell Sample
-##################
+   SMBus Shell Sample
```

Comment:
thanks for switching to the new system :)

FWIW I will probably override this description and the EDAC one as part of https://github.com/zephyrproject-rtos/zephyr/pull/62367 and https://github.com/zephyrproject-rtos/zephyr/pull/62352 : the description should not contain the word "sample", and is meant to read as if you were completing the sentence "This code sample shows how to ...", i.e. "Interact with SMBus peripherals using shell commands." in that case.

Thanks again, and sorry that I didn't see you added this to this PR and missed the opportunity to review before it was merged.

Reply:
OK, btw @kartben could you check why smbus sample is not included into: https://docs.zephyrproject.org/latest/hardware/peripherals/smbus.html ?

Reply:
It is :) The cache TTL for docs pages is 24 hours I think, so you were probably getting a cached version of the page.

Reply:
Ah, yes, now I see it, sorry.